src/ol/feature.js

```diff
@@ -302,13 +302,10 @@ ol.FeatureStyleFunction;
  * @return {ol.FeatureStyleFunction} A style function.
  */
 ol.Feature.createStyleFunction = function(obj) {
-  /**
-   * @type {ol.FeatureStyleFunction}
-   */
   var styleFunction;
 
   if (goog.isFunction(obj)) {
-    styleFunction = /** @type {ol.FeatureStyleFunction} */ (obj);
+    styleFunction = obj;
```

Comment:
The typing here and above makes sense to me. This is about making sure styleFunction is set to an ol.FeatureStyleFunction. Does the compiler complain about that when a specific flag is used?